### PR TITLE
NXP-27786: rename DocumentMultivaluedProperty.addItem to make it easi…

### PR DESCRIPTION
**Pull request by michaelva**
…er to find in studio and make it consistant with the name of the opposite remove operation

### CODE REVIEW

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedProperty.java [hunk 1]
```diff
@@ -41,10 +41,10 @@
...
+@Operation(id = AddEntryToMultiValuedProperty.ID, category = Constants.CAT_DOCUMENT, label = "Add entry into multi-valued metadata", description = "Add value into the field expressed by the xpath parameter. This field must be a multivalued metadata.<p> 'checkExists' parameter enables to add value only if doesn't already exists in the field: <ul><li> if checked, the value will not be added if it exists already in the list</li><li>if not checked the value will be always added</li</ul>.<p> Remark: <b>only works for a field that stores a list of scalars (string, boolean, date, int, long) and not list of complex types.</b></p><p>Save parameter automatically saves the document in the database. It has to be turned off when this operation is used in the context of the empty document created, about to create, before document modification, document modified events.</p>", aliases = { "DocumentMultivaluedProperty.addItem" })
```
Comment (efge) [inline modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedProperty.java:44]: You must keep the old alias as well.

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedProperty.java [hunk 2]
```diff
@@ -41,10 +41,10 @@
...
 public class AddEntryToMultiValuedProperty extends AbstractOperationMultiValuedProperty {
 
-    public static final String ID = "DocumentMultivaluedProperty.addItem";
+    public static final String ID = "Document.addEntrytoMultivaluedProperty";
```
Comment (efge) [inline modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedProperty.java:47]: Our convention for operation names is to have an uppercase after the period.

**Comment by efge** [comment]
Also please specify a fix version in the NXP ticket

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedProperty.java (hunk 1 above)
Comment (kevinleturc) [inline modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedProperty.java:44]: Are we confident to remove `AddEntryToMultivaluedProperty` alias?

File: modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedProperty.java (hunk 2 above)
Comment (kevinleturc) [inline modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedProperty.java:47]: The other operation is first letter capitalized, we may want to have the same thing:
```suggestion
    public static final String ID = "Document.AddEntrytoMultivaluedProperty";
```

**Comment by michaelva** [comment]
@efge and @kevinleturc  thanks for the reviews

**Comment by efge** [comment]
Don't forget to squash (it's better to squash before the reviews in this kind of situation)

**Comment by efge** [comment]
@michaelva the pushed commit message seems truncated. Too late to change it now though.

**Comment by michaelva** [comment]
@efge sorry for the commit message. I am not sure what I did wrong here.

**Pull request merged**